.github/workflows/regression_v2.yml

```diff
@@ -0,0 +1,703 @@
+# Regression Detection Suite
+#
+# This workflow runs under the following conditions:
+#   - Once per day, based on a schedule
+#   - On demand, triggered from https://github.com/vectordotdev/vector/actions/workflows/regression_v2.yml
+#
+# The workflow accepts two optional inputs:
+#   - The baseline SHA:
+#     - If not specified, the SHA from 24 hours ago on origin/master is used.
+#   - The comparison SHA:
+#     - If not specified, the current HEAD of origin/master is used.
+#
+# This workflow runs regression detection experiments, performing relative
+# evaluations of the baseline SHA and comparison SHA. The exact SHAs are determined
+# by how the workflow is triggered.
+#
+# The goal is to provide quick feedback on Vector's performance across a variety
+# of configurations, checking if throughput performance has degraded or become
+# more variable in the comparison SHA relative to the baseline SHA.
+#
+# Docker image tags are based on the resolved SHAs.
+
+name: Regression Detection Suite (new)
+
+on:
```

Comment:
Do you plan to keep the old regression workflow to be triggerable via `/ci-run-regression` on PRs as-needed? Or add that support to this workflow? Or drop it altogether?

Reply:
My goal is to eventually completely replace the existing `regression.yml` workflow with this one because `completely` is quite tricky to maintain. 

Great question about `/ci-run-regression`, I am also thinking of removing that completely. However, the SHA inputs can be used to trigger the suite on PRs on an even more fine-grained level, e.g. diff between individual commits on a given PR.

Reply:
👍 

> Great question about /ci-run-regression, I am also thinking of removing that completely. However, the SHA inputs can be used to trigger the suite on PRs on an even more fine-grained level, e.g. diff between individual commits on a given PR.

That's fair, this new workflow can be run manually for PRs, it is just a bit more of a tedious process to identify the commits to compare and you also lose the report being posted to the PR itself. It's up to you 🙂

Reply:
I will keep this in mind 👍  We can incrementally improve the new workflow by making it more easier to track results, publish nightly results etc.